vello/README.md

```diff
@@ -0,0 +1,296 @@
+<div align="center">
+
+# Vello
+
+**An experimental GPU compute-centric 2D renderer**
+
+[![Linebender Zulip](https://img.shields.io/badge/Linebender-%23gpu-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/stream/197075-gpu)
+[![dependency status](https://deps.rs/repo/github/linebender/vello/status.svg)](https://deps.rs/repo/github/linebender/vello)
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#license)
+[![wgpu version](https://img.shields.io/badge/wgpu-v0.20.0-orange.svg)](https://crates.io/crates/wgpu)
+
+[![Crates.io](https://img.shields.io/crates/v/vello.svg)](https://crates.io/crates/vello)
+[![Docs](https://docs.rs/vello/badge.svg)](https://docs.rs/vello)
+[![Build status](https://github.com/linebender/vello/workflows/CI/badge.svg)](https://github.com/linebender/vello/actions)
+
+</div>
+
+Vello is an experimental 2D graphics rendering engine written in Rust, with a focus on GPU compute.
+It can draw large 2D scenes with interactive or near-interactive performance, using [`wgpu`] for GPU access.
+
+Quickstart to run an example program:
+
+```shell
+cargo run -p with_winit
+```
+
+![image](https://github.com/linebender/vello/assets/8573618/cc2b742e-2135-4b70-8051-c49aeddb5d19)
+
+It is used as the rendering backend for [Xilem], a Rust GUI toolkit.
+
+> [!WARNING]
```

Comment:
We should change the formatting of this now, to use something which renders well on crates.io

That might just be using the ⚠️ emoji

Reply:
Updating readme files is intentionally out of scope and planned as follow-up work to reduce the review friction here.